js/OTPALayer.js

```diff
@@ -49,32 +49,35 @@ L.OTPALayer = L.FeatureGroup.extend({
 
     // When layer is added to map, also add LocationLayer
     // TODO: remove locationlayer when this layer is removed!
-    this._locationLayer = L.marker(self._location, {'draggable': true})
-        .on('dragend', function(e) {
-          self.setLocation(e.target._latlng); // UPDATES ISOCHRONE WHEN PIN MOVED
-        }).addTo(map);
+    this._locationLayer = new L.marker(self._location, {'draggable': true});
+    this._locationLayer.on('dragend', function(e) {
+      self.setLocation(self._locationLayer.getLatLng()); // UPDATES ISOCHRONE WHEN PIN MOVES
+    });
+    map.addLayer(this._locationLayer);
 
     var lastLayer = null;
     var onEachPoint = function(style) {
-        return function(feature, layer) {
-          layer.on({
-              mouseover: function highlightFeature(e) {
-                  var layer = e.target;
-
-                  if (lastLayer) {
-                      lastLayer.setStyle(style());
-                  }
-                  layer.setStyle(self._highlightedPointsetStyle());
-                  layer.bringToFront();
-
-                  self.fireEvent('select', {data: layer.feature.properties});
-              },
-
-              mouseout: function resetHighlight(e) {
-                  lastLayer = layer;
+      return;
```

Comment:
Whoops, didn't mean to commit that. Good catch. (Was trying to find a way to make the draggable marker not be super slow; seems to be related to [this](https://github.com/Leaflet/Leaflet/issues/2164).)